model_analyzer/entrypoint.py

```diff
@@ -152,16 +154,16 @@ def get_server_handle(config, gpus):
     return server
 
 
-def validate_triton_server_path(config):
+def validate_triton_path(config, key):
```

Comment:
Generalize this function so that `triton_server_path` and `triton_install_path` can both use it.